### PR TITLE
Improve machine pools when there are a large number of machines

### PR DESCRIPTION
**Pull request by nwmac**
Fixes rancher/dashboard#6142

When there are a large number of machines, its hard to see how many pools and machines you have.

This PR improves the presentation:

- Adds machine pool and machines counts to the cluster detail view (in the mast head)
- Adds a machine count and summary bar to each pool 

Also:

- Localise the masthead detail labels
- Update two formatters to respect the `dashIfEmpty` property on a column (ensures we show dashes for empty values when these formatters are used) 

Before:

image

After:

image

### CODE REVIEW

File: shell/detail/provisioning.cattle.io.cluster.vue [hunk 1]
```diff
@@ -564,15 +580,16 @@ export default {
...
                 </div>
                 <div v-else v-html="t('resourceTable.groupLabel.notInANodePool')">
                 </div>
                 <div v-if="group.ref && group.ref.nodeTemplate" class="description text-muted text-small">
                   {{ group.ref.providerDisplay }} &ndash;  {{ group.ref.providerLocation }} / {{ group.ref.providerSize }} ({{ group.ref.providerName }})
                 </div>
               </div>
-              <div v-if="group.ref" class="right">
+              <div v-if="group.ref" class="right group-header-buttons">
+                <MachineSummaryGraph :row="poolSummaryInfo[group.ref]" :horizontal="true" class="mr-20" />
```
Comment (richard-cox) [inline shell/detail/provisioning.cattle.io.cluster.vue:592]: This pages shows rke1 nodes or rke2 machines, so there's two lists here. In order to show the same graph for rke2 clusters the component just needs adding to the tab above (`<Tab v-if="showMachines"`)

**Review by richard-cox — CHANGES_REQUESTED** [review]
One small tweak required to show graph for RKE2 clusters

File: shell/detail/provisioning.cattle.io.cluster.vue (hunk 1 above)
Comment (richard-cox) [reply shell/detail/provisioning.cattle.io.cluster.vue:592]: i tried adding it in, but there's a minor styling issue
image

Comment (nwmac) [reply shell/detail/provisioning.cattle.io.cluster.vue:592]: What was the styling issue? I've updated, as per the suggestion - looks good to me.

Comment (richard-cox) [reply shell/detail/provisioning.cattle.io.cluster.vue:593]: I was seeing the plus button flush to the right side of the table

**Review by richard-cox — APPROVED** [review]
The failing e2e test has been removed in master

**Pull request merged**